drake/examples/HolonomicDrive/makeGrads.m

```diff
@@ -0,0 +1,59 @@
+function makeGrads(n)

```

Comment:
I think the whole thing fell apart when using structs of symbols, and that's why I rolled my own